tree/30_generic_methods/package_install_version_cmp_update.cf

```diff
@@ -165,19 +165,54 @@ bundle agent package_install_version_cmp_update(package_name, version_comparator
                         classes        => classes_generic_two("${old_class_prefix}", "${class_prefix}"),
                         action         => policy("${action}");
```

Comment:
This file should not be modified, we do not implement package management with the deprecated methods.